proto/src/main/proto/Rss.proto

```diff
@@ -234,6 +234,13 @@ message ShuffleCommitResponse {
   string retMsg = 3;
 }
 
+enum ServerStatus {
+  ACTIVE = 0;
```

Comment:
Should we add `UNKNOWN` for this enum?

Reply:
Protobuf has a default value named UNRECOGNIZED

Reply:
OK.